### PR TITLE
PIM-3622: Be notified when a mass edit is done

### PR DESCRIPTION
**Pull request by solivier**
| Q                    | A
| -------------------- | ---
| Bug fix?             | No
| New feature?         | Yes
| BC breaks?           | Yes
| CI currently passes? | Yes
| Tests pass?          | Yes (test for the new feature only)
| Scenarios pass?      | Yes
| Checkstyle issues?*  | No
| PMD issues?**        | No
| Changelog updated?   | No
| Fixed tickets        | PIM-3622
| DB schema updated?   | No
| Migration script?    | No
| Doc PR               | No

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -909,6 +909,12 @@ oro.locale:
 # Datagrid
 Selected products successfully deleted: Selected products successfully deleted
 
+pim_mass_edit:
+    notification:
+        mass_edit:
+            success: Mass edit finished
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/messages.en.yml]: "Mass edit successful" ?

Comment (solivier) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/messages.en.yml]: import/export notifications says 'finished"

File: features/notifications/display_notifications_when_mass_edit_jobs_are_finished.feature [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    And I am logged in as "Julia"
+
+  Scenario: Successfully display a notification when a mass-edit job is finished
+    Given a disabled "boat" product
+    And a disabled "jet-ski" product
+    And I am on the products page
+    When I mass-edit products boat and jet-ski
+    And I choose the "Change status (enable / disable)" operation
+    And I enable the products
+    And I wait 2 seconds
```
Comment (nidup) [inline features/notifications/display_notifications_when_mass_edit_jobs_are_finished.feature]: may we wait for the end of the execution as we do for import/export?

File: features/mass-action/change_status.feature [hunk 3]
```diff
@@ -5,7 +5,7 @@ Feature: Change status of many products at once
   I need to be able to change the status of many products at once
 
   Background:
-    Given the "default" catalog configuration
+    Given the "apparel" catalog configuration
```
Comment (jjanvier) [inline features/mass-action/change_status.feature]: we should use the catalog footwear whenever possible, not this one

**Pull request merged**